stannp/mock.go

```diff
@@ -86,7 +86,7 @@ func (mc *MockClient) SendLetter(_ *letter.SendReq) (*letter.SendRes, *util.APIE
 			Cost:    util.RandomString(10),
 			Created: util.RandomString(10),
 			Format:  util.RandomString(10),
-			Id:      0,
+			Id:      "0",
```

Comment:
Could we add test cases to verify it deserializes `Id: 0` as well?

Reply:
@z-cran added :+1: